packages/app/src/systems/Network/components/NetworkSelector/NetworkSelector.test.tsx

```diff
@@ -26,7 +26,7 @@ describe('NetworkSelector', () => {
     });
   });
 
-  it('should open dropdown with given networks', async () => {
+  it.skip('should open dropdown with given networks', async () => {
```

Comment:
Why do we need to skip these unit tests?

Reply:
I am ok skipping / removing those as they're already covered in e2e

Reply:
Oh, I'm going to re-check these tests